src/CollAction/Controllers/ProjectsController.cs

```diff
@@ -94,7 +94,7 @@ public ViewResult StartInfo()
         [HttpPost]
         [ValidateAntiForgeryToken]
         [Authorize]
-        public async Task<IActionResult> Create([Bind("Name,Description,Goal,Proposal,CreatorComments,CategoryId,LocationId,Target,End")] CreateProjectViewModel createProjectViewModel)
+        public async Task<IActionResult> Create([Bind("Name,Description,Goal,Proposal,CreatorComments,CategoryId,LocationId,Target,End,DescriptionVideoLink")] CreateProjectViewModel createProjectViewModel)
```

Comment:
It's fine to not use bind, but you don't "have" to use it if you have an appropiately restricted viewmodel. Might be a bit easier, just as long as you're sure you can handle a viewmodel with all parameters filled in.